src/amo/reducers/home.js

```diff
@@ -80,14 +106,29 @@ const reducer = (
   action: Action
 ): HomeState => {
   switch (action.type) {
+    case FETCH_HOME_ADDONS:
+      return {
+        ...state,
+        resultsLoaded: false,
+      };
+
     case LOAD_HOME_ADDONS: {
-      const { popularExtensions } = action.payload;
+      const {
+        featuredCollection,
+        popularExtensions,
+      } = action.payload;
 
       return {
         ...state,
+        featuredCollection: featuredCollection.results
+          .slice(0, LANDING_PAGE_ADDON_COUNT)
```

Comment:
I do this here because the [Collections API](https://addons-server.readthedocs.io/en/latest/topics/api/collections.html#list) does not allow to specify the number of results we want.

Reply:
Works but sounds like a bug worth filing: we could probably cut down on the response data size if we could send a `page_size` query.